include/yaml_cpp_catkin/yaml_cpp_fwd.hpp

```diff
@@ -0,0 +1,13 @@
+/**
+ * @file yaml_cpp_fwd.hpp
```

Comment:
Just a side note (no need to change if you prefer it like this):
When you don't specify a file name for the `@file` tag, it will default to this file. So if you just omit the filename here, the behaviour should be the same but it is easier to maintain (e.g. no need to adapt when a file is renamed, lower risk of copy-paste errors).